### ISSUE TITLE
misplaced reading system requirement

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2071]
The [collection-type definition](https://w3c.github.io/epub-specs/epub33/core/#sec-collection-type) introduces two values by making them a reading system requirement:

> When a scheme is not specified, Reading Systems SHOULD recognize the following collection type values:

We should obviously rephrase this to the values being defined by this specification, but I don't know what to do about the reading system requirement. What does it mean to recognize them?

**Comment by iherman** [comment]
> We should obviously rephrase this to the values being defined by this specification, but I don't know what to do about the reading system requirement. What does it mean to recognize them?

In the RS spec handling collection is OPTIONAL, and that is all we say. I think we should say that explicitly here.
